<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearGroup</class>
 <widget class="QWidget" name="GearGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>301</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>压力角</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_7">
      <item>
       <property name="text">
        <string>deg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rev</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rad</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>301</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>主齿轮中心半径</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_8">
      <item>
       <property name="text">
        <string>mm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>inch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ft</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>主齿轮齿数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>副齿轮齿数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>301</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>副齿轮中心半径</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_9"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_9">
      <item>
       <property name="text">
        <string>mm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>inch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ft</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>302</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>副齿轮齿宽</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_10"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_10">
      <item>
       <property name="text">
        <string>mm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>inch</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>302</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>模数</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_11"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_11">
      <item>
       <property name="text">
        <string>mm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>inch-1</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>302</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>主齿轮齿宽</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_12"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_12">
      <item>
       <property name="text">
        <string>mm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>inch</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
